its_the_way.py

```diff
@@ -0,0 +1,24 @@
+
+import os
+
+
+def file_start(path :str,start:str):
+    """
+    the function get a path of a directory and find the files in the directory that start with start
```

Comment:
Nice documentation, but please fix the indentation in this line.

Reply:
How should it look?